docs/blog/2021-11-02-our-ci-setup.md

```diff
@@ -143,15 +143,15 @@ We're guessing we're not the only ones who feel that way.
 ![take my money meme](/img/take-my-money-meme.jpg)
 
 ## Conclusion and takeaway
-Figuring this out was pretty hard. It took months of work. But, I think we are nearing CI nirvana.  Our builds are very reliable,
+Figuring this out was pretty hard. It took months of work. But, I think we are nearing CI Nirvana.  Our builds are very reliable,
```

Comment:
Not a proper noun, we aren't talking about the band here.  Youll notice its lower case in the wiki article https://en.wikipedia.org/wiki/Nirvana